packages/account/src/Components/leave-confirm/leave-confirm.tsx

```diff
@@ -70,10 +80,11 @@ export const TransitionBlocker = ({ dirty, onDirty }) => {
     });
 
     const leave = () => {
-        const { pathname } = next_location;
-
-        unblock();
-        history.push(pathname);
+        if (next_location && 'pathname' in next_location) {
```

Comment:
will ` if (next_location?. pathname)`  work here?

Reply:
Yes it will